charms/kfp-ui/tests/unit/test_relation_components.py

```diff
@@ -125,14 +124,14 @@ def __init__(self, *args):
         self.charm_executor = CharmReconciler(self)
```

Comment:
Should we change it to `charm_reconciler` here as well? Same as in charm code.

Reply:
good catch! changed